src/components/academies/AcademyContentCard.jsx

```diff
@@ -109,6 +109,14 @@ const AcademyContentCard = ({
     }
     return showAllOcmCourses;
   };
+
+  const toggleButtonText = (showMoreBtnEnabled, contentType, title, contentLength) => {
+    if (showMoreBtnEnabled) {
+      return title === 'Self-paced courses' ? `< Show less ${title}` : `< Show less ${title} courses`;
+    }
+    return title === 'Self-paced courses' ? `Show more ${title} (${contentLength}) >` : `Show more ${title} courses (${contentLength}) > `;
```

Comment:
we can pass just 'Self-paced' and 'Executive Education' as a title in "renderableContent" to make this logic more simple and more readable.

Reply:
@jajjibhai008 We cannot change the title since it is displayed in other locations, such as the header.
<img width="1440" alt="Screenshot 2024-04-18 at 7 20 41 PM" src="https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/113524403/47178cd6-6c40-40ab-87ea-0221b053e74b">